controllers/controller.js

```diff
@@ -64,4 +64,12 @@ exports.getUsers=(req,res)=>{
     return selectUsers().then((userArr)=>{
         res.status(200).send({users: userArr})
     })
+}
+
+exports.deleteComment=(req,res,next)=>{
+    const id=req.params.comment_id
+    return deleteCommentByID(id).then(()=>{
+        res.status(204).send()
```

Comment:
Like the choice of status code and that you've left send empty as we arn't looking to send anything back on the body.